src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,52 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    public static final int MAXIMUM_STORAGE_SIZE = 10;
+    private final K[] keys;
+    private final V[] values;
+    private int size;
+
+    public StorageImpl() {
+        this.keys = (K[]) new Object[MAXIMUM_STORAGE_SIZE];
+        this.values = (V[]) new Object[MAXIMUM_STORAGE_SIZE];
+        this.size = 0;
```

Comment:
```suggestion
        keys = (K[]) new Object[MAXIMUM_STORAGE_SIZE];
        values = (V[]) new Object[MAXIMUM_STORAGE_SIZE];
```
by default int variables have 0 value
you should use this only in case for example when you initialize keys or values variable using passed parameters to the constructor and parameters have the same name, so it helps to distinguish fields and parameters

Reply:
fixed